README.md

```diff
@@ -17,12 +17,12 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
+[X] Strings
+[X] Booleans
 [] Undefined
 [] NaN
-[] Integers
-[] Arrays
+[X] Integers
```

Comment:
JavaScript only has `Number`.  It does not define different types of numbers, like integers, short, long, floating-point etc.